platform/fabric/services/state/recipients.go

```diff
@@ -192,8 +191,28 @@ func RespondRequestRecipientIdentity(context view.Context) (view.Identity, error
 
 // ExchangeRecipientIdentitiesView models the view of the initiator of an exchange of recipient identities.
 type ExchangeRecipientIdentitiesView struct {
-	Network string
-	Other   view.Identity
+	Network       string
```

Comment:
ditto

Reply:
I needed this to be able to choose a particular identity. In the specific case an HSM identity.

Reply:
Look at the IOU sample